BookingsApi/BookingsApi.DAL/BookingsDbContext.cs

```diff
@@ -57,6 +57,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     }
                 }
             }
+
+            modelBuilder.Entity<JusticeUser>().HasQueryFilter(u => !u.Deleted);
```

Comment:
This wouldn't enable you you restore deleted users going forward if the query filters it out to begin with. I thought we agreed restored users will be enabling the existing row and new rows were reserved for new unique users

Reply:
If we need to query deleted justice users, we can override that filtering by adding `.IgnoreQueryFilters()`.

eg `var allUsersIncludingDeleted = await context.JusticeUsers.IgnoreQueryFilters().ToList();`

That will remove the the where `!u.Deleted` from the query.